test/finsupp_notation.lean

```diff
@@ -12,14 +12,49 @@ example : (fun₀ | 1 | 2 | 3 => 3 | 3 => 4) 2 = 3 := by
 example : (fun₀ | 1 | 2 | 3 => 3 | 3 => 4) 3 = 4 := by
   simp
 
+section repr
+
+/-- info: fun₀ | 1 => 3 | 2 => 3 -/
+#guard_msgs in
+#eval (Finsupp.mk {1, 2} (fun | 1 | 2 => 3 | _ => 0) (fun x => by aesop))
+
 /--
 info:
+fun₀
+| ["there are five words here", "and five more words here"] => 5
+| ["there are seven words but only here"] => 7
+| ["just two"] => 2
```

Comment:
Indentation doesn't match between the `#eval` and the `#check` command, can you fix that?

Reply:
Which version do you prefer?

Reply:
I suppose the two-space-indented version is a bit clearer, but either works for me.

Reply:
Done.